docs/developer/stock-allocation.mdx

```diff
@@ -128,6 +128,37 @@ the [`updateWarehouse`](developer/api-reference/mutations/update-warehouse.mdx)
 There is an option to change the following warehouse fields: `name`, `slug`, `email`, `address` and
 also define privacy and `click and collect` options.
 
+The warehouse could be defined as private or public. When the warehouse is public, the stock
+is shown, otherwise it isn't.
+
+### Click-and-collect warehouses
+
+The `click and collect` options allowed defining the warehouse as a collection point.
+Three options can be used:
+
+- `local stock` - the stock from this warehouse is used during the fulfillment process,
+  this option is only available for public warehouses
+- `all warehouses` - the global stock for a given product is used during the fulfillment
+  process, the available quantity is not restricted by the quantity of this warehouse,
+  and ordered products can be shipped from a different warehouse
```

Comment:
"from a different warehouse" - any other warehouse? Or only from those with "all warehouses" setting? We could specify.

Reply:
Any other that is in a checkout channel. I'll specify this.